engine/rpcserver.go

```diff
@@ -694,16 +664,23 @@ func (s *RPCServer) GetAccountInfoStream(r *gctrpc.GetAccountInfoRequest, stream
 			log.Errorln(log.DispatchMgr, pipeErr)
 		}
 	}()
+	init := make(chan struct{}, 1)
+	init <- struct{}{}
 
 	for {
```

Comment:
🐒 🥃  Cute aside, but can be ignored.
I kept wondering how to localise this into the for scope.
[This works](https://go.dev/play/p/TsAraTqUm-Y).
Shame we can't close the channel.
Don't think it's any better or worse to try to send to a full channel than to try to read from an empty one.
```suggestion
	for init := make(chan struct{}, 1); ; {
```

Reply:
Localizing it within the for scope is a good idea, but the current approach is simpler and easier to understand.
So, I'll ignore it.